client/src/components/Footer.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Col, PageHeader } from 'react-bootstrap';
+
+const Footer = () => (
+  <div>
+    <Col xs={12} md={12}>
+      <div>
+        <PageHeader className="home-section-page-footer">Privacy Policy</PageHeader>
```

Comment:
The content of this component is PrivacyPolicy specific.  
Perhaps we could pull the PrivacyPolicy file  into its own full page(styled off of `info.scss` styles) and the Footer could link to that page(add the '/privacy-policy' route to router.js).